src/audiences/edit/components/estimate.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { getEstimate } from '../data';
+import { Sparklines, SparklinesLine } from 'react-sparklines';
+
+const { __ } = wp.i18n;
+
+const Estimate = props => {
+	const {
+		audience,
+		title,
+		sparkline = true,
+	} = props;
+
+	const [ loading, setLoading ] = useState( true );
+
+	const [ estimate, setEstimate ] = useState( {
+		count: 0,
+		total: 0,
+		histogram: [],
+	} );
+
+	const fetchEstimate = () => {
+		( async () => {
+			const estimateResponse = await getEstimate( audience );
+			setEstimate( estimateResponse );
+			setLoading( false );
+		} )();
+	};
+
+	useEffect( fetchEstimate, [ audience ] );
```

Comment:
I'm thinking it might be a bit less confusing to use a class component here, given we have a few different elements of state and some side effects